src/include/duckdb/execution/operator/persistent/buffered_csv_reader.hpp

```diff
@@ -99,6 +99,8 @@ struct BufferedCSVReaderOptions {
 	std::map<LogicalTypeId, StrpTimeFormat> date_format = {{LogicalTypeId::DATE, {}}, {LogicalTypeId::TIMESTAMP, {}}};
 	//! Whether or not a type format is specified
 	std::map<LogicalTypeId, bool> has_format = {{LogicalTypeId::DATE, false}, {LogicalTypeId::TIMESTAMP, false}};
+	//! Whether the file should be overridden when query fails
```

Comment:
should this be overwritten instead of overridden?
Also, I think the option should prolly be named more on the line of:
stream_write